src/handlers.rs

```diff
@@ -85,9 +85,9 @@ pub fn get_room(room_id: &str) -> Result<Response, Rejection> {
     let pool = &storage::MAIN_POOL;
     let conn = pool.get().map_err(|_| Error::DatabaseFailedInternally)?;
     // Get the room info if possible
-    let raw_query = format!("SELECT id, name FROM {} where id = (?1)", storage::MAIN_TABLE);
+    let raw_query = format!("SELECT id, name, pinned_message_id FROM {} where id = (?1)", storage::MAIN_TABLE);
```

Comment:
This needs to be rebased on `dev`, and then this should be
```
let raw_query = "SELECT id, named, pinned_messages_id FROM main WHERE id = ?";
```
(I also explicitly dropped the obvious 1, and removed the superfluous brackets).

Reply:
Also looking back, pinned_messages_id probably doesn't need to be returned here